config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
> Yep, I think two-fer should be the second core exercise

ten four. I'll make the change!

> Is that any different to having them be unlocked_by: hello-world?

Slightly. If they're unlocked_by: null then you get them as soon as you submit hello world. If they are unlocked by hello world, then you have to go through the completion ritual first, before you get them.

We should probably have a few of each.